### PR TITLE
Update slides.html

### PR DESCRIPTION
**Pull request by subinamehta**
Added speaker notes

### CODE REVIEW

File: topics/metagenomics/tutorials/metatranscriptomics/slides.html [hunk 1]
```diff
@@ -354,13 +354,22 @@
 
 ![functional analysis workflow schematic](../../images/metatranscriptomics/workflow_functional.png)
 
+???
+HUMAnN2 
+- next generation
+- HMP Unified Metabolic Analysis Network
+- developed by Huttenhower lab,
```
Comment (bgruening) [inline topics/metagenomics/tutorials/metatranscriptomics/slides.html]: , is too much

File: topics/metagenomics/tutorials/metatranscriptomics/slides.html [hunk 2]
```diff
@@ -529,19 +562,27 @@
...
+cellulose 1,4 beta-cellulobiosidase-> hydrolysis of cellulose
+upregulation
 ---
 
 ## Functions associated with a selected taxon
 
 .image-75[![results](../../images/metatranscriptomics/results_functions_taxon.png)]
 
+???
+gene abundance-coprothermobacter and clostridium. here we see the glycolysis pathway showing same trend
```
Comment (bgruening) [inline topics/metagenomics/tutorials/metatranscriptomics/slides.html]: @subinamehta can you put a little bit more effort here, to make those proper sentences and such? Thanks!

Comment (subinamehta) [reply topics/metagenomics/tutorials/metatranscriptomics/slides.html]: OK..I will add the explanations 👍

**Comment by shiltemann** [comment]
thanks a lot @subinamehta!

**Pull request merged**